### PR TITLE
N36: CyberGate

### PR DESCRIPTION
**Pull request by cybersoulK**
Part of #1098

### CODE REVIEW

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
The general content of this post looks okay, but there's a lot of issues;

* You've created a new file, rather than adding to [the existing one](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/content/news/036/index.md).
* The links are not formatted correctly (if you look at the [preview](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/258c79fc4cd732277a76a5f4d3c4cb9bc6a8161a/content/posts/newsletter-036/index.md) they're all showing up with extra markup.
* There's quite a few formatting issues (extra new lines in the list, placeholder brackets left in the initial description, etc)

Could you please update this PR/resubmit to resolve these issues and move your content to the right file?

**Comment by cybersoulK** [comment]
is everything good now? @17cupsofcoffee

**Comment by ozkriff** [comment]
:construction:

**Review by ozkriff — APPROVED** [review]
Fixed the formatting, made a few style tweaks  to fit with the rest of the newsletter, and reduced the image size

**Comment by ozkriff** [comment]
I've rephrased a few things. @cybersoulK are you ok with my changes?

File: content/news/036/index.md [hunk 1]
```diff
@@ -88,6 +88,33 @@ include:
...
 [steamworks]: https://crates.io/crates/steamworks
 
+### [CyberGate][cybergate-yt]
+
+![hundreds of colliding colored balls in the air](entities.png)
+_The server and clients are able to smoothly handle
+over a thousand balls rained from above_
+
+CyberGate ([YouTube][cybergate-yt], [Discord][cybergate-dis]) by CyberSoul
+is a new multiplayer project that aims at procedurally generating destinct
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:98]:
```suggestion
is a new multiplayer project that aims at procedurally generating distinct
```

File: content/news/036/index.md [hunk 2]
```diff
@@ -88,6 +88,33 @@ include:
...
+Recent updates:
+
+- Bandwidth became 16 times smaller by implementing entity prioritization
+  \+ other techniques.
+- Interpolation and Jitter prediction makes entities way smoother.
+- Automatic and Reliable Spawn and Despawn of entities.
+- Many other features and optimizations to do with rapier 3d physics,
+  wgpu renderer and quinn (quic) protocol.
+
+Please, [join our Discord server][cybergate-dis] to participate in tests.
```
Comment (17cupsofcoffee) [inline content/news/036/index.md:111]:
```suggestion
[Join the Discord server][cybergate-dis] to participate in tests.
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**